react/features/base/config/middleware.js

```diff
@@ -125,12 +125,6 @@ function _setConfig({ dispatch, getState }, next, action) {
         }));
     }
 
-    if (action.config.disableSelfView) {
```

Comment:
Removing this causes a problem when we check if we should hide self-view or not. Mentioned in a comment below

Reply:
This in general is wrong, you set it to true and then there is no way to set it to false, and if UI is hidden the only way is to edit localstorage.